### ISSUE TITLE
RFE: make resolvectl control manager settings

### ISSUE THREAD

**Issue by yuwata** [systemd/systemd#8916]
### Submission type

 - Request for enhancement (RFE)

Currently, `resolvectl` can only control link settings. It may be useful that `resolvectl` can also controls manager settings.

**Comment by bugfood** [comment]
Meanwhile, is there any way to do this via busctl, or is there any other workaround? In #11318 (closed as a duplicate of this bug), I had trouble with that too.

**Comment by yuwata** [comment]
@bugfood No, `resolved` has no interface to change global parameter yet.
So, please edit /etc/systemd/resolved.conf.
